crates/mun_codegen/src/mock.rs

```diff
@@ -20,6 +20,9 @@ impl salsa::Database for MockDatabase {
     fn salsa_runtime(&self) -> &salsa::Runtime<MockDatabase> {
         &self.runtime
     }
+    fn salsa_runtime_mut(&mut self) -> &mut salsa::Runtime<MockDatabase> {
```

Comment:
```suggestion
    }

    fn salsa_runtime_mut(&mut self) -> &mut salsa::Runtime<MockDatabase> {
```

Reply:
Why this change? If rust format doesnt catch it, does it matter?

Reply:
Consistency. Everywhere we are formatting it this way. Only these couple of added functions don't, which stood out.